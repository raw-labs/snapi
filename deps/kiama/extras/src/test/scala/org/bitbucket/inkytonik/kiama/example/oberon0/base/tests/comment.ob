(* A comment *)
MODULE comment;
   (* A multi
     line comment
       doo dee doo *)
(* nesting (* in comments *) is fun *)
BEGIN
   (* a bit more (*
      multi line 
      nest (* and again *)
      *)*)
   (*(**)*)
   (**)
END(*a comment in line*)comment.
  (*
   a comment at the end
  *)
