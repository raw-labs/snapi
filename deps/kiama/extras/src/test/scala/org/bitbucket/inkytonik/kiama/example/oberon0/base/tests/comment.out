* _pp.ob
MODULE comment;

BEGIN
    
END comment.
[6]* c
[6]#include <stdio.h>
[6]int main (int argc, char *argv[0]) {
[6]    ;
[6]    return 0;
[6]}
